exercises/atbash-cipher/AtbashCipherExample.m

```diff
@@ -0,0 +1,61 @@
+#import "AtbashCipherExample.h"
+
+static NSDictionary<NSString *, NSString *> *cipherDictionary;
+
+@implementation AtbashCipher
+
++ (void)initialize {
+    if (self == [AtbashCipher class]) {
+        cipherDictionary = [self createCipherDictionary];
+    }
+}
+
++ (NSDictionary<NSString *, NSString *> *)createCipherDictionary {
+    NSArray<NSString *> *alphabet = @[@"a", @"b", @"c", @"d", @"e", @"f", @"g", @"h", @"i", @"j", @"k", @"l", @"m", @"n", @"o", @"p", @"q", @"r", @"s", @"t", @"u", @"v", @"w", @"x", @"y", @"z"];
+    NSMutableDictionary<NSString *, NSString *> *cipherDict = [[NSMutableDictionary alloc] initWithCapacity:26];
+    
+    NSUInteger count = alphabet.count;
+    
+    for (int i = 0; i < count; i++) {
+        NSString *key = alphabet[i];
+        NSString *value = alphabet[count - i - 1];
+        
+        cipherDict[key] = value;
+    }
+
+    return cipherDict;
+}
+
++ (NSString *)encode:(NSString *)input {
+    input = [input lowercaseString];
```

Comment:
In C++ these are usually marked as`conts`, similarly to Swift defaults as let for params. My preference would be to leave the warning on because of the C++ and Swift way.  Perhaps the best way to prevent this is to make the param constant too. 

stackoverflow.com/questions/17228334/what-is-the-best-way-to-create-constants-in-objective-c